### PR TITLE
Fix/sonar

### PR DESCRIPTION
**Pull request by marikaris**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits
- [ ] Added Feature/Fix to release notes

### CODE REVIEW

File: molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/controller/MappingServiceController.java [hunk 1]
```diff
@@ -985,7 +988,12 @@ public void saveCategoryMapping(
   }
 
   /**
-   * Generate algorithms based on semantic matches between attribute tags and descriptions
+   * Generate algorithms based on semantic matches between attribute tags and descriptions <<<<<<<
+   * HEAD
+   *
+   * <p>=======
+   *
+   * <p>>>>>>>> f2109293178f2f7f60abeab99a29e019e74016d6
```
Comment (fdlk) [inline molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/controller/MappingServiceController.java]: Needs merge

File: molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/controller/MappingServiceController.java [hunk 2]
```diff
@@ -456,7 +458,11 @@ public String cloneMappingProject(@RequestParam("mappingProjectId") String mappi
    * searchTerms will be used directly in the SemanticSearchService. If the searchTerms are not
    * defined by users, it will use the ontologyTermTags in the SemantiSearchService. If neither of
    * the searchTerms and the OntologyTermTags exist, it will use the information from the
-   * targetAttribute in the SemanticSearchService
+   * targetAttribute in the SemanticSearchService <<<<<<< HEAD
+   *
+   * <p>=======
+   *
+   * <p>>>>>>>> f2109293178f2f7f60abeab99a29e019e74016d6
```
Comment (fdlk) [inline molgenis-semantic-mapper/src/main/java/org/molgenis/semanticmapper/controller/MappingServiceController.java]: Merge

File: molgenis-semantic-search/src/main/java/org/molgenis/semanticsearch/string/NGramDistanceAlgorithm.java [hunk 3]
```diff
@@ -17,19 +17,32 @@
...
  *
+ * <p>=======
+ *
+ * <p>
+ *
+ * <p>How to use? LevenShteinDistanceModel model = new LevenShteinDistanceModel(2); double
+ * similarityScore = model.stringMatching("Smoking", "Smoker", false);
+ * System.out.println(similarityScore);
+ *
+ * <p>>>>>>>> f2109293178f2f7f60abeab99a29e019e74016d6
```
Comment (fdlk) [inline molgenis-semantic-search/src/main/java/org/molgenis/semanticsearch/string/NGramDistanceAlgorithm.java]: merge

File: molgenis-semantic-search/src/main/java/org/molgenis/semanticsearch/string/NGramDistanceAlgorithm.java [hunk 4]
```diff
@@ -17,19 +17,32 @@
  * stop words has been defined, in the method stringMatching(), one of the parameters
  * "removeStopWords" indicates whether the stop words will be used to remove the useless or
  * meaningless words from the String. This the stop words could be customized by
- * setStopWords(List<String> stopWords) or setStopWords(String[] stopWords).
+ * setStopWords(List<String> stopWords) or setStopWords(String[] stopWords). <<<<<<< HEAD
+ *
```
Comment (fdlk) [inline molgenis-semantic-search/src/main/java/org/molgenis/semanticsearch/string/NGramDistanceAlgorithm.java]: merge

**Comment by dennishendriksen** [comment]
Superseded by #7988

**Pull request closed without merging**